src/main/scala/org/constellation/domain/healthcheck/HealthCheckLoggingHelper.scala

```diff
@@ -109,22 +117,45 @@ object HealthCheckLoggingHelper {
 
   // Logging and metrics
   def logHealthcheckConsensusDecicion[F[_]: Sync](
-    decision: HealthcheckConsensusDecision,
+    decision: HealthcheckConsensusDecision)(
+    metrics: Metrics,
```

Comment:
why currying if it's not used?

Reply:
Separation of data (decision) and services that are used to do something with that data (metrics, logger). I'm not using currying, but I could and I think it looks cleaner.